### PR TITLE
updated styling on restaurant homepage

### PR DESCRIPTION
**Pull request by jillianrramirez**
still need to fix links in navbar, restaurant logo on slide one, and paragraph spacing and styling

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	font-size: 80px;
+	text-shadow: 1px 1px 5px #000;
+}
+
+#story h3 {
+	font-family: 'Raleway', sans serif;
+	font-weight: 700;
+	font-size: 26px;
+	margin-top: 0;
+	margin-bottom: 0;
```
Comment (rockygg) [inline css/styles.css]: this margin-bottom: 0; goes on the h1 element

**Comment by rockygg** [comment]
hey this looks great! you can go ahead and fix those things you mentioned in your comment then push to this branch again, this pull request will automatically update just let me know when you make those changes before you merge in

**Comment by jillianrramirez** [comment]
Hey Rocky! I fixed issues and pushed my changes. Let me know if the push went through and it's okay to merge this branch.

**Comment by rockygg** [comment]
👍

**Pull request merged**